src/Makefile

```diff
@@ -93,3 +93,7 @@ build: kokkoskernels-build-lib
 
 install: kokkoskernels-install
  
+clean: kokkoskernels-clean kokkos-clean
```

Comment:
Do we also need to clean kokkos ?

Reply:
@kyungjoo-kim maybe not, I had put that in there while I was making sense of the issue and did not retest without once I had a better idea what was going on. Would you like me to test without and update the PR?

Reply:
Sounds good, thanks @kyungjoo-kim !